### ISSUE TITLE
About the training and testing data

### ISSUE THREAD

**Issue by alalba221** [AaronJackson/vrn#126]
Hi Aaron,
you said in #12 that you didn't train the network on HELEN and LFPW , so I trained the network on IBUG  and AFW using 6000 images for training and about 800 for testing. Here is the loss a got:
epoch 0 : train_loss 0.352367, eval_loss 0.387095
epoch 1 : train_loss 0.278556, eval_loss 0.420478
epoch 2 : train_loss 0.239114, eval_loss 0.385385
epoch 3 : train_loss 0.211572, eval_loss 0.436594
epoch 4 : train_loss 0.190551, eval_loss 0.464007
epoch 5 : train_loss 0.175974, eval_loss 0.484641
epoch 6 : train_loss 0.163320, eval_loss 0.493853
epoch 7 : train_loss 0.153328, eval_loss 0.565732
epoch 8 : train_loss 0.145184, eval_loss 0.531872
epoch 9 : train_loss 0.138747, eval_loss 0.520749
epoch 10 : train_loss 0.132051, eval_loss 0.595860
epoch 11 : train_loss 0.126847, eval_loss 0.558690
epoch 12 : train_loss 0.121628, eval_loss 0.688304
epoch 13 : train_loss 0.117325, eval_loss 0.581210
epoch 14 : train_loss 0.113913, eval_loss 0.597701
epoch 15 : train_loss 0.109812, eval_loss 0.629373
epoch 16 : train_loss 0.107197, eval_loss 0.650522
epoch 17 : train_loss 0.103911, eval_loss 0.615461
epoch 18 : train_loss 0.100986, eval_loss 0.663053
epoch 19 : train_loss 0.098672, eval_loss 0.642940
epoch 20 : train_loss 0.096295, eval_loss 0.654145
epoch 21 : train_loss 0.094171, eval_loss 0.625737
epoch 22 : train_loss 0.091768, eval_loss 0.762728
epoch 23 : train_loss 0.089807, eval_loss 0.760316
epoch 24 : train_loss 0.088324, eval_loss 0.612928
epoch 25 : train_loss 0.085970, eval_loss 0.817148
epoch 26 : train_loss 0.084619, eval_loss 0.792491
epoch 27 : train_loss 0.083374, eval_loss 0.687534

as the training loss decreasing, the testing loss keeps increasing. I am not sure if it is because I didn't adopt the Flips images as training data or something else.
So would you tel me which folder you used for training and which for testing?
Thanks

**Comment by AaronJackson** [comment]
I can't remember exactly. The exact sets we used are described in the paper. If your testing error is going up, perhaps you have generated the two sets of volumes with different scales or something.
